index.d.ts

```diff
@@ -1,4 +1,3 @@
-import * as stream from 'stream'
 import * as http from 'http'
 
 type HTTPMethods = 'DELETE' | 'delete' |
```

Comment:
This one actually might cause a bit of a conflict with vanilla typing for many http clients as many just take a `string` for this

Reply:
This PR did not change this. Am I correct?

Reply:
Yep. I just wanted to let you know. If you want me to change it, just say so. Otherwise you can always leave changing this for later or not at all.

Reply:
Thanks, that's ok as it is